### PR TITLE
guard views/dashboard by `tugraz_authenticated`

### PR DESCRIPTION
**Pull request by martinobersteiner**

### CODE REVIEW

File: invenio_theme_tugraz/webpack.py [hunk 1]
```diff
@@ -17,9 +17,10 @@
             entry={
                 "invenio-theme-tugraz-theme": "./less/invenio_theme_tugraz/theme.less",
                 "invenio-theme-tugraz-js": "./js/invenio_theme_tugraz/theme.js",
+                "invenio-theme-tugraz-unlock": "./js/invenio_theme_tugraz/unlock.js",
             },
             dependencies={
-                # add any additional npm dependencies here...
+                "jquery": "~3.2.1",  # zammad-form, semantic-ui's modals
```
Comment (utnapischtim) [inline invenio_theme_tugraz/webpack.py:23]: did you checked which jquery version InvenioRDM is using?

File: setup.cfg [hunk 2]
```diff
@@ -62,6 +62,8 @@ invenio_assets.webpack =
     invenio_theme_tugraz_theme = invenio_theme_tugraz.webpack:theme
 invenio_config.module =
     invenio_theme_tugraz = invenio_theme_tugraz.config
+invenio_base.finalize_app =
```
Comment (utnapischtim) [inline setup.cfg:65]: please add a separate commit for the finalize_app change

File: invenio_theme_tugraz/webpack.py (hunk 1 above)
Comment (martinobersteiner) [reply invenio_theme_tugraz/webpack.py:23]: I did
at the time I wrote the code, this version was the correct one
now, `invenio_previewer` has moved on to `"^3.3.1"`, whereas other invenio-modules are still at `"^3.2.1"`

changed to `"^3.2.1"`, which should fit with current invenio

File: setup.cfg (hunk 2 above)
Comment (martinobersteiner) [reply setup.cfg:65]: done, see PR #323

**Review by utnapischtim — APPROVED** [review]

**Pull request merged**